tests/cypress/support/index.js

```diff
@@ -18,3 +18,8 @@ import '../../../lib/index';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+beforeEach(() => {
+  Cypress.Cookies.defaults({
+    preserve: /^wordpress.*?/,
+  });
+});
```

Comment:
Can you share this technique to https://10up.github.io/Open-Source-Best-Practices/testing/#e2e-testing?

Reply:
Also the @10up/cypress-wp-setup :)

Reply:
@dinhtungdu Raised https://github.com/10up/Open-Source-Best-Practices/pull/80 and https://github.com/10up/cypress-wp-setup/pull/14